h/accounts/schemas.py

```diff
@@ -377,6 +377,10 @@ class CreateUserAPISchema(JSONSchema):
                 'type': 'string',
                 'format': 'email',
             },
+            'display_name': {
+                'type': 'string',
+                'maxLength': 30,
```

Comment:
We currently have the length limits for usernames and display names duplicated in multiple places in the code. There are some constants at the top of `h/models/user.py`. Could we use those here?